diff --git a/src/WebServer_404.ino b/src/WebServer_404.ino
index 4a5520f4..80715890 100644
--- a/src/WebServer_404.ino
+++ b/src/WebServer_404.ino
@@ -6,11 +6,11 @@
 void handleNotFound() {
   checkRAM(F("handleNotFound"));
 
-  if (wifiSetup)
-  {
-    WebServer.send(200, F("text/html"), F("<meta HTTP-EQUIV='REFRESH' content='0; url=/setup'>"));
-    return;
-  }
+  //if (wifiSetup)
+  //{
+   // WebServer.send(200, F("text/html"), F("<meta HTTP-EQUIV='REFRESH' content='0; url=/setup'>"));
+  //  return;
+ // }
 
   if (!isLoggedIn()) { return; }
 
diff --git a/src/WebServer_RootPage.ino b/src/WebServer_RootPage.ino
index a7df588e..0debeb84 100644
--- a/src/WebServer_RootPage.ino
+++ b/src/WebServer_RootPage.ino
@@ -9,11 +9,11 @@ void handle_root() {
   checkRAM(F("handle_root"));
 
   // if Wifi setup, launch setup wizard
-  if (wifiSetup)
-  {
-    WebServer.send(200, F("text/html"), F("<meta HTTP-EQUIV='REFRESH' content='0; url=/setup'>"));
-    return;
-  }
+  //if (wifiSetup)
+  //{
+  //  WebServer.send(200, F("text/html"), F("<meta HTTP-EQUIV='REFRESH' content='0; url=/setup'>"));
+  //  return;
+ // }
 
   if (!isLoggedIn()) { return; }
   navMenuIndex = 0;
